resources/lang/fr/common.php

```diff
@@ -27,7 +27,7 @@
     'airline'        => 'Compagnie',
     'subfleet'       => 'Sous-Flotte',
     'distance'       => 'Distance',
-    'fuel'           => 'Carburant',
+    'fuel'           => 'Fuel',
```

Comment:
I would've kept "Carburant" since fuel is not a french word and "carburant" is the literal translation of fuel

Reply:
Understood I'm not a French speaker, so unsure why he did that. But I can ask.  Thank you !!!

Reply:
OK, Fuel is common in French, but "carburant" is prettier.
I agree with arthurpark06

Reply:
Ici je sais pas trop, perso j'aime pas carburant, trop général, mais ce n'est que mon avis.
Je n'aime pas trop fuel non plus

Reply:
Ouais mais le truc c'est que fuel c'est pas français donc quelqu'un de mauvais en anglais il comprendra pas... Essence ce serait pas top non plus, kérozène peut être ? pas extra mais bon...